src/sage/sets/recursively_enumerated_set.pyx

```diff
@@ -1574,9 +1554,9 @@ def search_forest_iterator(roots, children, algorithm='depth'):
          [0, 1, 2], [0, 2, 1], [1, 0, 2], [1, 2, 0], [2, 0, 1], [2, 1, 0]]
     """
     # Little trick: the same implementation handles both depth and
-    # breadth first search. Setting position to -1 makes a depth search
+    # breadth first search. Setting position to -1 initiates a depth search
     # (you ask the children for the last node you met). Setting
-    # position on 0 makes a breadth search (enumerate all the
+    # position on 0 initiates a breadth search (enumerate all the
```

Comment:
This change is not quite correct as it doesn't actually start the computation (that is done after this is set). "makes" is correct here.

Reply:
OK, I thought it was obvious that the computation is started later, and "makes" simply sounds wrong to my (non-native speaker) ears.

Would "initiates the ... search" be more correct?

Reply:
It is not inconceivable that setting a variable initializes a computation. It is an unlikely situation that someone misinterprets, but it is technically wrong.

"make" is being used somewhere in between "creating" and "forcing" here. Perhaps "initializes"? Although I wouldn't change it.

Reply:
OK, then "initializes the subsequent ... search"?

Reply:
That's good with me.

Reply:
Done.